### ISSUE TITLE
Loading GraphQL documents fails with undefined as JSX generic type argument.

### ISSUE THREAD

**Issue by geon** [dotansimha/graphql-code-generator#1773]
**Describe the bug**
This code can't be used in the same file as a gql-tag:
```ts 
<Foo<undefined> />;
```

**To Reproduce**

In my-file.tsx, have this code:

```ts 
<Foo<undefined> />;
```

Run `gql-gen --config graphql-codegen.yml`

It will fail with the message

```
  ✔ Parse configuration
  ❯ Generate outputs
    ❯ Generate src/client/graphql/generated-client-types.ts
      ✔ Load GraphQL schemas
      ✖ Load GraphQL documents
        → Unexpected token (1:4)
        Generate


 Found 1 error

  ✖ src/client/graphql/generated-client-types.ts
    SyntaxError: Unexpected token (1:4)
```

However, changing my-file.tsx to another type as the generic argument works:

```ts 
<Foo<string> />;
```

The type `null`has the same problem:

```ts 
<Foo<null> />;
```

As a workaround, I have defined a type alias:

```ts 
type Undefined = undefined;
<Foo<Undefined> />;
```

1. My GraphQL schema:

Should not be relevant.

2. My GraphQL operations:

Should not be relevant.

3. My `codegen.yml` config file:

```yml
schema:
  - "src/server/graphql/schema.ts"
require: "ts-node/register"
overwrite: true
config:
  immutableTypes: true
generates:
  src/client/graphql/generated-client-types.ts:
    documents: #"src/client/**/*.{ts,tsx}"
      - "my-file.tsx"
    plugins:
      - add: "//tslint:disable"
      - "typescript"
      - "typescript-operations"
```

**Expected behavior**

Loading GraphQL documents should work the same, regardless of the type arguments given to JSX generics.

**Environment:**

- OS: Mac OS 10.14.3 (18D109)
- `@graphql-codegen/...`: ^1.1.1
- NodeJS: v10.12.0

**Comment by dotansimha** [comment]
We are using `graphql-tag-pluck` to discover operations from code file. @DAB0mB what do you think? Maybe it's a Babel configuration?

**Comment by DAB0mB** [comment]
@dotansimha just released a new version. PRs should be created by `renovate` and should be merged in `graphql-toolkit` and codegen. Do you want me to add tests to `codegen` as well?

**Comment by dotansimha** [comment]
@geon can you please try `1.1.2-alpha-036fb85a.17`?

**Comment by dotansimha** [comment]
Fixed in 1.1.3 🎉
